docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Design
+
+The system is implemented as an k8s operator using the [operator-sdk](https://github.com/operator-framework/operator-sdk)
+but is deployed as a DaemonSet instead of Deployment with [filtering](https://github.com/operator-framework/operator-sdk/blob/master/doc/user/event-filtering.md) only events for the DaemonSet pod node.
+
+There are two controllers one for [Node](https://godoc.org/k8s.io/api/core/v1#Node)
+and the other for the CRD NodeNeworkState.
+
+## Components
+
+- [Node Controller](developer-guide-node.md) - developer guide and design of `Node` handler.
+- [NodeNetworkState Controller](developer-guide-state.md) - developer guide and design of `NodeNetworkState` handler.
+- [NodeNetworkConfigurationPolicy Controller](developer-guide-policy.md) - developer guide and design of `NodeNetworkConfigurationPolicy` handler.
+
+# Development
+
+[Development guide](developer-guide-commands.md) is a go-to reference point for
+development helper commands, building, testing, container images and local
+cluster.
+
+# Open a Pull Request
+Kubernetes-nmstate generally follows the [standard github pull request process](https://gist.github.com/Chaser324/ce0505fbed06b947d962), but there is a layer of additional specific differences:
+
+kubernetes-nmstate specific github workflow:\
+The first difference you'll see is that a bot will begin applying structured labels to your PR.
+
+The bot may also make some helpful suggestions for commands to run in your PR to facilitate review.
+These /command options can be entered in comments to trigger auto-labeling and notifications. Refer to its command reference documentation.
+
+Common new contributor PR issues are:
+
+* Missing DCO sign-off:
+Developers Certificate of Origin (DCO) Sign-off is a requirement for getting patches into the project (see [Developers Certificate of Origin](https://developercertificate.org/)).\
```

Comment:
You don't need to enforce new line using `\`. Could can use normal paragraph and indent the following lines (at least I believe so)